### ISSUE TITLE
Exporting a pack with overrides causes manifest.json to not list any mods

### ISSUE THREAD

**Issue by bluebear94** [gorilla-devs/GDLauncher#555]
**Describe the bug**
In a pack exported to the Twitch format with any files selected to be included in the export causes the `manifest.json` file in the generated pack to not list any mods.

**To Reproduce**
1. Create a new Forge instance
2. Add some mods, any mods
3. Create a file `hi.txt` in the instance folder
4. Export the pack with `hi.txt` selected to be included in the generated pack
5. Open the generated .zip file
6. Open `manifest.json` in the archive

**Expected behavior**
The `files` field in `manifest.json` should list the mods that were selected, rather than being empty as it is right now.

**Operating System:**
 - OS: Linux (Manjaro)
 - Version 1.0.8

**Comment by ChaseIngebritson** [comment]
Unable to reproduce on v1.0.8.

![2020-07-01 16_11_27-overrides](https://user-images.githubusercontent.com/11217065/86292092-b34f1300-bbb5-11ea-9cb8-52c35ca11cfb.png)
![2020-07-01 16_12_00-● manifest json (deleted) - Visual Studio Code  Administrator](https://user-images.githubusercontent.com/11217065/86292097-b518d680-bbb5-11ea-9676-41a3ada46e6d.png)

When you open the instance manager for the instance you wish to export, do you see the Twitch logo by the mods that you added to the instance? I believe this is what indicates that the mod has a Project ID, File ID, etc. to be able to pull from the API once it's imported somewhere else.

![2020-07-01 16_13_34-GDLauncher](https://user-images.githubusercontent.com/11217065/86292319-2b1d3d80-bbb6-11ea-8d6d-6aaf53b423f9.png)

In the example above, only CTM-MC and BiomesOPlenty will be added to the manifest.

**Operating System:**
* OS: Windows
* Version 10

**Comment by bluebear94** [comment]
> When you open the instance manager for the instance you wish to export, do you see the Twitch logo by the mods that you added to the instance?

Yes, I do. I don't see any interesting output in the console, either.

**Comment by bluebear94** [comment]
Also note that this bug doesn't occur if I don't select any overrides for the generated pack.

**Comment by Kevadroz** [comment]
Also happens to me. However, the cause may diverge slightly, i remember exporting two times with the same export options and it only happened on one, i also may not remember it perfectly, ¿maybe has something to do with external files combined with curse files? Still, i have to export a modpack with no overrides to get the mod list, and selecting no overrides causes another bug, exporting the entire instance as if i selected everything, however that is a bug for another github issue.

My Specs:
OS: Manjaro KDE (linux 64bit)
GDL Version: 1.0.8
